src/script/conversation/ClientMismatchHandler.ts

```diff
@@ -84,27 +85,31 @@ export class ClientMismatchHandler {
    * @note Contains clients of which the backend is sure that they should not be recipient of a message and verified they no longer exist.
    * @param recipients User client map containing redundant clients
    * @param payload Payload of the request
-   * @param [conversationEntity] Conversation entity
+   * @param conversationEntity Conversation entity
```

Comment:
```suggestion
   * @param [conversationEntity] Conversation entity
```
Conversation entity is still optional.

Reply:
Yes, but no need to reflect that in the JSDoc since we are using TypeScript here.

Reply:
At least in VS Code, adding square brackets around the parameter name is considered invalid in TypeScript:
![screenshot](https://user-images.githubusercontent.com/5497598/79544724-b98e2280-808f-11ea-95f2-21bc80dacae3.png)

While removing the brackets highlights the parameter name correctly:
![screenshot](https://user-images.githubusercontent.com/5497598/79544728-ba26b900-808f-11ea-9ee7-6d624364889a.png)

Reply:
Ok, looks like TSDoc has no support for optionals: https://github.com/Microsoft/vscode/issues/66079 & https://github.com/Microsoft/tsdoc/issues/151#issuecomment-473423780